tests/Recordings/e2e_code_generation_create_delete.json

```diff
@@ -0,0 +1,249 @@
+            "Access-Control-Max-Age": "1728000",
+            "CF-Space-Id": "t7rprcaoexcq",
+            "Content-Type": "application\/vnd.contentful.management.v1+json",
+            "Date": "Fri, 06 Oct 2017 17:41:07 GMT",
+            "ETag": "\"57fc70b47bcb4283e8a06d9e5ed9650e\"",
+            "Server": "Contentful",
+            "Strict-Transport-Security": "max-age=15768000",
+            "X-Content-Type-Options": "nosniff",
+            "X-Contentful-Request-Id": "183ae202815a3760e0b8399b302288c0",
+            "Connection": "keep-alive",
+            "Set-Cookie": "visid_incap_673446=4nJmyjcKSYad+gm6m4JnRjPA11kAAAAAQUIPAAAAAADANM+GWhTHORbrTUYx0yqM; expires=Sat, 06 Oct 2018 07:07:22 GMT; path=\/; Domain=.contentful.com, nlbi_673446=BmcPcYstdmLyVsfE6lKYhQAAAAAuMynwV3INhVj9h52SlIy0; path=\/; Domain=.contentful.com, incap_ses_417_673446=zkFfeHtb5wzBpHKAX3zJBTPA11kAAAAA35eZAPJu+pICIquRPQESNA==; path=\/; Domain=.contentful.com",
+            "X-Iinfo": "10-35019627-35019638 NNNN CT(105 108 0) RT(1507311666969 121) q(0 0 3 -1) r(4 4) U5",
+            "X-CDN": "Incapsula",
+            "Content-Encoding": "gzip",
+            "Transfer-Encoding": "chunked"
+        },
+        "body": "{\n  \"sys\": {\n    \"space\": {\n      \"sys\": {\n        \"type\": \"Link\",\n        \"linkType\": \"Space\",\n        \"id\": \"t7rprcaoexcq\"\n      }\n    },\n    \"id\": \"5WzuLOwjFCUYWSsCaIMO64\",\n    \"type\": \"Entry\",\n    \"createdAt\": \"2017-10-06T17:41:07.521Z\",\n    \"updatedAt\": \"2017-10-06T17:41:07.521Z\",\n    \"createdBy\": {\n      \"sys\": {\n        \"type\": \"Link\",\n        \"linkType\": \"User\",\n        \"id\": \"4Q3e6duhma7V6czH7UXHzE\"\n      }\n    },\n    \"updatedBy\": {\n      \"sys\": {\n        \"type\": \"Link\",\n        \"linkType\": \"User\",\n        \"id\": \"4Q3e6duhma7V6czH7UXHzE\"\n      }\n    },\n    \"version\": 1,\n    \"contentType\": {\n      \"sys\": {\n        \"type\": \"Link\",\n        \"linkType\": \"ContentType\",\n        \"id\": \"author\"\n      }\n    }\n  },\n  \"fields\": {\n    \"name\": {\n      \"en-US\": \"Lee Adama\"\n    },\n    \"location\": {\n      \"en-US\": {\n        \"lon\": 50,\n        \"lat\": 50\n      }\n    },\n    \"isActive\": {\n      \"en-US\": true\n    },\n    \"misc\": {\n      \"en-US\": {\n        \"codename\": \"Apollo\"\n      }\n    },\n    \"picture\": {\n      \"en-US\": {\n        \"sys\": {\n          \"type\": \"Link\",\n          \"linkType\": \"Asset\",\n          \"id\": \"24jR8tPh6cWyQyWecs8USO\"\n        }\n      }\n    }\n  }\n}\n"
+    }
+},{
+    "request": {
+        "method": "GET",
+        "url": "https:\/\/api.contentful.com\/spaces\/t7rprcaoexcq\/assets\/24jR8tPh6cWyQyWecs8USO",
+        "headers": {
+            "Host": "api.contentful.com",
+            "User-Agent": "GuzzleHttp\/6.2.1 curl\/7.54.0 PHP\/7.1.8",
+            "X-Contentful-User-Agent": "sdk contentful-management.php\/\/0.9.0-dev; platform PHP\/7.1.8; os macOS;",
```

Comment:
just a stupid observation: Does this mean that if you change the version, this test will fail, because the expected request was not fired?

Reply:
No, the user agent string is saved for debugging purposes, but in `tests/bootstrap.php` it is specified that it should not be used when checking for a match 🙂

Reply:
👌